### DISCUSSION TITLE
Add config for GuzzleHttp/Client

### DISCUSSION THREAD

**Discussion by ghost** [westacks/telebot#76] [Ideas]
Hello! I suggest adding the following:
1) in config/telebot.php add:
```php
'http' => [
    'config' => [
        'verify' => false,
    ],
],
```

2) in src/Telebot.php, change line 89 to the following:
`$this->client = new Client([ 'http_errors' => false ] + $config['http']['config']);`

This way we will get rid of the error: **cURL error 60: SSL certificate problem: self-signed certificate in certificate chain**

Thanks.

**Comment by punyflash** [comment]
This seems reasonable, however, better to use `array_merge` than `+` operator, so default library parameters could be overwritten. Also, there is no need to add 2nd nesting layer to array - what's in config already is a config, nothing else will be there, so just use:
```php
'http' => [
    'verify' => false,
],
```
And add backward compatibility check in case `$config['http']` is not set:
```php
$this->client = new Client(array_merge([ 'http_errors' => false ], $config['http'] ?? []));
```
  You are free to make pull request

**Reply by ghost** [reply]
> This seems reasonable, however, better to use `array_merge` than `+` operator, so default library parameters could be overwritten. You are free to make pull request

Okay, but in any case, to solve my problem, I edit your library files myself. I understand that this is not correct. so I created this request with a suggestion to make new parameters in config/telebot.php

**Reply by punyflash** [reply]
It's fine, you could've just created a pull request. I'll add this after the next telegram update in case no one does, but notice, that your idea will not be credited in that case.
